src/pages/clusters/Detail/Basic/index.tsx

```diff
@@ -146,6 +162,19 @@ export default function Basic(props: any) {
         >
           {intl.formatMessage({ id: 'pages.clusterDetail.basic.edit' })}
         </Button>
+        {
+          version === pipelineV2 || (
+            <Tooltip title={intl.formatMessage({ id: 'pages.message.cluster.upgrade.tooltip' })}>
+              <Button
```

Comment:
Why not using microapp? It's a button for internal usage.

Reply:
fixed